Dockerfile

```diff
@@ -1,7 +1,25 @@
 FROM python:3.8-buster
 
 WORKDIR /pyfstat
+
+# Ephemerides arguments
+## Put ephemerides in this folder
+ARG EPHEMERIDES_FOLDER=ephemerides/
```

Comment:
Actually, I'm not sure of this relative path:

WORKDIR ensures that we are referring to /pyfstat/ephemerides, but once spawned, LALPULSAR_DATADIR will be "ephemerides/" only (relative path)